internal/k8s/logging_kubectl_runner.go

```diff
@@ -0,0 +1,80 @@
+package k8s
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io"
+	"io/ioutil"
+
+	"github.com/windmilleng/tilt/internal/logger"
+)
+
+// wraps a kubectlRunner with logging
+type loggingKubectlRunner struct {
+	logLevel logger.Level
+	runner   kubectlRunner
+}
+
+var _ kubectlRunner = loggingKubectlRunner{}
+
+func (k loggingKubectlRunner) logExecStart(ctx context.Context, args []string, stdin io.Reader) (newStdin io.Reader, err error) {
```

Comment:
have you considered interface to take stdin as bytes or string instead of a Reader?

i feel like all the callers pass in a string, and it would make the implementation substantially simpler

Reply:
yep, much better!